From 98633858cdcbeb5646d830839ab66d3274b8d45a Mon Sep 17 00:00:00 2001
From: Neal Liu <neal_liu@aspeedtech.com>
Date: Tue, 30 Sep 2025 10:10:35 +0800
Subject: [PATCH] configs: ibex-ast2700: enable ASPEED_OTP support

Enable ASPEED_OTP configure.

Signed-off-by: Neal Liu <neal_liu@aspeedtech.com>
Change-Id: Id02fbfcc7ca2812aa3d1a02d05615e98239239bd
(cherry picked from commit 046dccb25832ae877d3bacac621d20c18338cbc2)
Signed-off-by: Vasanthakumar K <vasanthakumark@ami.com>
---
 configs/ibex-ast2700_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/ibex-ast2700_defconfig b/configs/ibex-ast2700_defconfig
index 3a1beee56393..50f3fcfa864b 100644
--- a/configs/ibex-ast2700_defconfig
+++ b/configs/ibex-ast2700_defconfig
@@ -134,6 +134,7 @@ CONFIG_SCSI=y
 CONFIG_DM_SCSI=y
 CONFIG_SYS_NS16550=y
 CONFIG_SYS_NS16550_MEM32=y
+CONFIG_ASPEED_OTP=y
 # CONFIG_TIMER is not set
 CONFIG_UFS=y
 CONFIG_ASPEED_UFS=y
-- 
2.34.1

